taskcluster/translations_taskgraph/parameters.py

```diff
@@ -45,7 +45,7 @@ def get_defaults(_):
                 "training-teacher": {
                     "disp-freq": "1",
                     "save-freq": "25",
-                    "valid-freq": "50",
+                    "valid-freq": "49",
```

Comment:
This should be reverted, validation runs correctly once in a regular CI run

Reply:
I only set this to force-trigger a training task.

I explicitly added a `[TRASHME]` in the commit message to avoid merging the PR with this commit